src/components/Checkbox/Checkbox.scss

```diff
@@ -47,76 +32,108 @@
             display: flex;
             align-items: center;
         }
+
+        &:hover {
+            .Checkbox__box {
+                background: var(--color-background-selected-subtlest-hover, #CCE4FF);
+            }
+        }
+
+        &:active {
+            .Checkbox__box {
+                background: var(--color-background-selected-subtlest-pressed, #99CAFF);
+            }
+        }
     }
 
     &__svg {
         position: absolute;
-        top: 0;
+        top: var(--border-width-thick);
         left: 0;
         fill: none;
-        stroke: var(--color-background);
+        fill-rule:evenodd;
+        clip-rule:evenodd;
+        stroke: var(--color-background-information-subtlest-default);
         stroke-width: 2;
         stroke-linecap: round;
         stroke-linejoin: round;
     }
 
+    &--disabled {
+        .Checkbox__label {
+            .Checkbox__box {
+                background: var(--color-background-selected-subtlest-disabled, #E6F2FF);
+                border-color: var(--color-border-disabled, #CCC); 
+
+                &:focus {
+                    box-shadow: none;
+                }
+            }
+        }
+    }
+
     &__input {
-        display: none;
+        width: 0;
+        height: 0;
```

Comment:
it looks like specifying width and height instead of using `display: none` led to the extra space 🤔 the best you could see it for ThreeCheckboxState. Is it intended? 
![Uploading Screenshot 2024-07-19 at 16.47.02.png…]()

Reply:
nice catch!
note: I changed the display option so that keyboard navigation is possible, and you can use `TAB` to reach the checkbox.

Reply:
Thank you for the explanation and making changes. 
I tested it in several browsers and it looks good only in Chrome to me. 
Firefox:
<img width="313" alt="Screenshot 2024-07-22 at 14 56 22" src="https://github.com/user-attachments/assets/6cf3197f-26a3-4c74-b794-5ddf0faa9889">

Safari:
<img width="254" alt="Screenshot 2024-07-22 at 14 56 58" src="https://github.com/user-attachments/assets/046ce827-b230-4089-beca-e74be5b82afe">

Could you please have a look?

Not sure what is happening in Microsoft Edge, though 🫣

Reply:
Nice catch again! 
And here I was, thinking those differences are a problem of the past. I learned [something](https://developer.mozilla.org/en-US/docs/Web/CSS/appearance) new today.  From the look of this attribute, I think it should be ok in Edge too
I managed to use Browserstack to test Edge too and it looks good. but when I tried to record it, it all goes confused (I think it is browserstack). I can PM you the credentials if you want.